app/controllers/merchant/items_controller.rb

```diff
@@ -1,5 +1,20 @@
 class Merchant::ItemsController < Merchant::BaseController
-
+  #all merchant items
   def index
+    @merchant = current_user
+  end
+
+  def update
+    @merchant = current_user
+    @item = Item.find(params[:id])
+    if @item.active == true
```

Comment:
Nice, `@item.active` is already a boolean so I think `@item.active` would be more semantic.